scripts/migrate_score_override.py

```diff
@@ -0,0 +1,138 @@
+import itertools
+from django.contrib.auth.models import User
+from django.utils.timezone import now
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from openassessment.assessment.models import Assessment, AssessmentPart, StaffWorkflow
+from openassessment.workflow.models import AssessmentWorkflow, AssessmentWorkflowStep
+from student.models import anonymous_id_for_user
+from submissions.models import Score, ScoreAnnotation, Submission
+
+old_scores = Score.objects.filter(submission__isnull=True, reset=False)
+for score in old_scores:
+    points_override = score.points_earned
+    points_possible = score.points_possible
+
+    # Update old override with new reset
+    score.points_earned = 0
+    score.points_possible = 0
+    score.reset = True
+    score.save()
+
+    #Grab latest submission with matching student_item
+    submission = Submission.objects.filter(student_item=score.student_item)[:1].get()
+
+    # Create new staff override score
+    new_score = Score(
+        student_item=score.student_item,
+        submission=submission,
+        points_earned=points_override,
+        points_possible=points_possible,
+    )
+    new_score.save()
+
+    # Fetch staff id from score course for ScoreAnnotation
+    course_id = SlashSeparatedCourseKey.from_deprecated_string(score.student_item.course_id)
```

Comment:
SlashSeparatedCourseKey has been deprecated by upstream albeit ahead of our merge point. ( https://github.com/edx/edx-platform/pull/15617 )

Maybe you can future-proof this with something like:

from opaque_keys.edx.keys import CourseKey
course_id = CourseKey.from_string(score.student_item.course_id)